src/plugins/recordTypes/person/fields.js

```diff
@@ -1259,31 +1270,35 @@ export default (configContext) => {
                 },
               },
             },
-            suppliedBirthDate: {
+            suppliedStructuredBirthDateGroup: {
               [config]: {
+                dataType: DATA_TYPE_STRUCTURED_DATE,
                 messages: defineMessages({
                   fullName: {
-                    id: 'field.persons_common.suppliedBirthDate.fullName',
-                    defaultMessage: 'Birth date supplied',
+                    id: 'field.persons_common.suppliedStructuredBirthDateGroup.fullName',
+                    defaultMessage: 'Supplied structured birth date',
```

Comment:
We don't normally put "structured" in the label for structured dates. "Supplied birth date" should suffice.

Reply:
I was trying to find a way to differentiate between the `suppliedBirthDateGroup` and the `suppliedStructuredBirthDateGroup` in the search with this (previously they both had shared `Birth date supplied`). Maybe the message for the group should be different in that case? I'll look around to see if there are any fields similar to this to model after.

Reply:
@ray-lee thoughts on adding a fullName to the groups for maker supplied info that would read `Maker supplied birth date`, `Maker supplied birth place`, etc. It's really only under adding the group conditions for searches but seems to make it a bit clearer imo.